MovingBlocks/Terasology
# Save screenshots regularly

### Contains

We wanted more game preview images, we got it!

### How to test

Just need to create a new game or load saved one. Then play and check that game takes screenshots regularly, and you can see them in detailed screens.

### Outstanding before merging

* [x]  **Replays** take screenshots and saves them to `saves` dir.
